Makefile

```diff
@@ -113,6 +114,7 @@ $(GOPATH)/bin/%:
 	@echo "Building $@"
 	@mkdir -p $(@D)
 	@docker run -i \
+		--user=$(UID)
 		-v $(abspath vendor/github.com/golang/protobuf):/opt/gopath/src/github.com/golang/protobuf \
```

Comment:
note that you need a continuation "\" and the end of the line in both places, that is why the "docker run" thing is complaining